Sample Applications/WPFGallery/Views/Samples/UserDashboardPage.xaml.cs

```diff
@@ -32,8 +32,38 @@ public UserDashboardPage(UserDashboardPageViewModel viewModel)
 
         private void EditButton_Click(object sender, RoutedEventArgs e)
         {
+            var command = (sender as Button)?.Command;
+            var commandParameter = (sender as Button)?.CommandParameter;
+            if (command != null && command.CanExecute(commandParameter))
+            {
+                command.Execute(commandParameter);
+            }
 
+            save_button.Focus();
         }
 
+        private void SaveButton_Click(object sender, RoutedEventArgs e)
+        {
+            var command = (sender as Button)?.Command;
+            var commandParameter = (sender as Button)?.CommandParameter;
+            if (command != null && command.CanExecute(commandParameter))
+            {
+                command.Execute(commandParameter);
+            }
+
+            edit_button.Focus();
+        }
+
+        private void CancelButton_Click(object sender, RoutedEventArgs e)
+        {
+            var command = (sender as Button)?.Command;
```

Comment:
Do we need to handle command execution here ? Does it not happen automatically ?